polygeist/tools/cgeist/Lib/Attributes.cc

```diff
@@ -40,13 +40,28 @@ static void addToPassThroughAttr(NamedAttribute &PassThroughAttr,
   std::vector<mlir::Attribute> Vec =
       PassThroughAttr.getValue().cast<ArrayAttr>().getValue().vec();
 
-  // TODO: find a way to add the attributes only if one does not exist already,
-  // and keep the list in sorted order.
```

Comment:
We've fixed the first condition in the comment, but not the second, right?

Reply:
No need to do now, but could keep comment (relevant part)

Reply:
Correct I can add a comment.